aiida/engine/processes/calcjobs/manager.py

```diff
@@ -113,7 +114,7 @@ def _get_jobs_from_scheduler(self):
             # Update the last update time and clear the jobs cache
             self._last_updated = time.time()
             jobs_cache = {}
-            self.logger.info('AuthInfo<{}>: successfully retrieved status of active jobs'.format(self._authinfo.pk))
+            self.logger.info('AuthInfo<%s>: successfully retrieved status of active jobs', self._authinfo.pk)
```

Comment:
Is there a reason to switch to the `%s` string interpolation instead of `{}` and `.format`?

Reply:
This makes the evaluation of the arguments to the `info` call lazily evaluated. This is useful when those operations are potentially costly and there is a chance that the log will be discarded anyway. Using the lazy evaluation is the [preferred method](https://docs.python.org/3/howto/logging.html#optimization) for logging for this reason, even though in this case the effect will be minimal

Reply:
I'm not sure that's the point. Anyway python needs to evaluate the argument before passing it to the `.info()` method, and the `.info()` method cannot do anything to defer it. The only way to do it would be if the method were to accept a callable and then decides if/when to call it, but this cannot be done with properties like `pk` (unless you return a lambda returning the value, but anyway I'm not sure the logger accepts lambdas or callables). 
I think the point of the link is not on whether to use `%s` vs. `format`, but rather to wrap the logging call into an `if` block `checking `logger.isEnabledFor(CURRENT_LEVEL)`.
For this reason (since anyway here we don't need to speed this up, if `pk` is slow we've got a problem anyway :-) ) I would suggest consistency in the code and using `format` as we do (almost?) everywhere else.

Reply:
Example proving the lambda concept:
```python
import time

def slow():
    time.sleep(2)
    return "slow return"

def myprint(value, shouldprint=True):
    print('inside myprint, before printing')

    # Potential checks if I need to print here
    if not shouldprint:
        return

    # Lazy evaluation
    if callable(value):
        value = value()

    print("Value: {}".format(value))
    print('inside myprint, after printing')


print("*"*72)

print("*** Fast computations")
# Everything fast
myprint("A")

print("*"*72)

print("*** Slow evaluation *before* calling myprint")
# Slow evaluation, then three messages printed quickly
myprint(slow())

print("*"*72)

print("*** Slow evaluation deferred inside myprint")
# First print occurs quickly, then value is slowly evaluated inside the method,
# then final two messages are printed
myprint(lambda: slow())

print("*"*72)

print("*** Should be slow")
t = time.time()
# Does not print but it's slow
myprint(slow(), shouldprint=False)
print("Time spent: {} s".format(time.time() - t))

print("*"*72)

print("*** Should be fast")
t = time.time()
# Does not print and returns quickly
myprint(lambda: slow(), shouldprint=False)
print("Time spent: {} s".format(time.time() - t))

print("*"*72)
```

Reply:
Agreed, I misread the link I gave. I'm happy with reverting the change, but then we probably want to add a global ignore rule for the warning that pylint will start to throw. They advise to pass string formatting arguments as arguments to the logging call to delay interpolation to last moment possible. Do you want me to revert?

Reply:
I think this exception makes sense only if you are doing this inside a loop that is very fast (I would say microseconds) where interpolating the string is slow. For us it does not matter so much I believe. There are also various discussions online, also on how from py3.6 there are even faster ways to do this with f-strings, see e.g. [here](https://github.com/PyCQA/pylint/issues/2395).

My suggestion would be to revert and add an ignore for this (does it even make sense to put a global ignore in the pylintrc for this error, in AiiDA?)

Reply:
I would say that if we are not going to use it, we should just put the ignore in the `.pylintrc` instead of littering the code with inline ignores